create/application_test.go

```diff
@@ -411,7 +411,7 @@ func TestApplicationWait(t *testing.T) {
 				app.Status.AtProvider.Hosts = []apps.VerificationStatus{{Name: "host.example.org"}}
 				app.Status.AtProvider.CNAMETarget = "some.target.example.org"
 				app.Status.AtProvider.BasicAuthSecret = &meta.LocalReference{Name: basicAuth.Name}
-				if err := apiClient.Status().Update(ctx, app); err != nil {
```

Comment:
Not sure what is going on with the fake client but updating the status subresource does not do anything but just calling `Update()` seems to update both the spec and the status. So the tests are now passing with just not using `Status().Update` anymore.

Reply:
super weird 😅